src/rpg_database/character_sheet/Focus.java

```diff
@@ -0,0 +1,57 @@
+package rpg_database.character_sheet;
+
+public class Focus {
+
+	private boolean isImproved;
+	private int value;
+	private Focuses focus;
+
+	public Focus(Focuses focus) {
+		this(focus, false);
+	}
+
+	public Focus(Focuses focus, boolean isFocusImproved) {
+		if (isFocusImproved) {

```

Comment:
Ez lehet gányolósan fog hangzani de szerintem egyszerűbb és olvashatóbb ternary operátorral az értékadás. a külön boolean konstansokkal való értékadásokat meg nem is értem:
```java
value = isFocusImproved ? 3 : 2;
isImproved = isFocusImproved;
```
esetlegesen a magic numbereket ki lehet emelni ebből konstanssá és akkor:

```java

private static final int FOCUS_VALUE_DEFAULT = 2;
private static final int FOCUS_VALUE_IMPROVED = 3;

.
.
.

value = isFocusImproved ? FOCUS_VALUE_IMPROVED : FOCUS_VALUE_DEFAULT ;
isImproved = isFocusImproved;
```

Reply:
Refactorálva.